service-api/module/Application/src/Model/Service/Pdfs/Service.php

```diff
@@ -167,8 +167,13 @@ private function getPdfStatus(Lpa $lpa, $type)
 
             // If we get here it exists in the bucket...
             return self::STATUS_READY;
-        } catch (\Aws\S3\Exception\S3Exception $ignore) {
-            $this->getLogger()->error('Exception while attempting to get PDF info from S3');
+        } catch (\Aws\S3\Exception\S3Exception $exception) {
+            if ($exception->getStatusCode() !== 404) {
+                $this->getLogger()->error('Exception while attempting to get PDF info from S3', [
+                    'error_code' => 'PDF_S3_HEAD_FAILED',
```

Comment:
Not for this ticket, but it could be useful to extract any magic strings we're using in logging context arrays into a central location so we can explain what they mean.